spec/requests/articles/publications_spec.rb

```diff
@@ -19,4 +18,15 @@
       expect(Article.last.published).to be true
     end
   end
+
+  describe "DELETE /publications" do
+    before do
+      article = Article.create! valid_attributes.merge!(published: true)
```

Comment:
I think it's better to create article in the `let` variable.
And I think we can create an article ourselves instead of sending request to the articles controller in the previous spec (on line 13 `post articles_path, params: { article: valid_attributes }`).